backend/alembic/versions/a8b9c0d1e2f3_add_linked_group_id_to_tasks.py

```
@@ -0,0 +1,633 @@
+            task_id = row[0]
+            try:
+                task_json = json.loads(row[1]) if isinstance(row[1], str) else row[1]
+                is_group_chat = task_json.get("spec", {}).get("is_group_chat")
+                if is_group_chat is True:
+                    conn.execute(
+                        sa.text(
+                            "UPDATE tasks SET is_group_chat = 1 WHERE id = :task_id"
+                        ),
+                        {"task_id": task_id},
+                    )
+            except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)